aws/src/main/java/nl/nn/adapterframework/filesystem/AmazonS3FileSystem.java

```diff
@@ -657,6 +675,11 @@ public void setProxyPort(Integer proxyPort) {
 		this.proxyPort = proxyPort;
 	}
 
+	/** S3 Storage Class, for more information see https://aws.amazon.com/s3/storage-classes/ */
+	public void setStorageClass(String storageClass) {
```

Comment:
There are many different storage classes; to get some kind of validation during the configuration I would like to see here a translation from the string to the enum value `StorageClass`.